Sources/RxComposableArchitecture/Internal/Deprecated.swift

```diff
@@ -7,6 +7,15 @@
 
 import Darwin
 
+// MARK: - Deprecated after 0.50.4
+
+@available(
+  *,
+  deprecated,
+  message: "Use 'Effect<Action>.Send' instead."
+)
+public typealias Send<Action> = Effect<Action>.Send
+
```

Comment:
please try to remove this (I don't think we have any usage of send yet)

Reply:
this one is from https://github.com/pointfreeco/swift-composable-architecture/pull/1911
it was also used internally by some core like store etc. so this one for also the warning if the codes use Send (if we pulling another core codes later on, we can see warning and change it into new one) wdyt gan? @jeffersonsetiawan

Reply:
If we remove this code and there is no error, I think we can go to remove it dik, as Toko is not using this Send<Action> code yet CMIIW